clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -2035,6 +2035,24 @@ void CIRGenModule::emitTopLevelDecl(Decl *decl) {
         emitTopLevelDecl(childDecl);
     break;
   }
+  case Decl::PragmaComment: {
+    const auto *PCD = cast<PragmaCommentDecl>(decl);
+    switch (PCD->getCommentKind()) {
+    case PCK_Unknown:
+      llvm_unreachable("unexpected pragma comment kind");
+    case PCK_Linker:
+      assert(!MissingFeatures::emitModuleLinkOptions() && "NYI");
```

Comment:
Btw, I added support for this couple weeks ago in MLIR LLVM dialect, could be a good next improvement.